backend/WP/Gutenberg/js/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-react"]
```

Comment:
The default pragma is React.createElement https://babeljs.io/docs/en/babel-preset-react, so I will use this configuration in order to compile to wp.element.createElement. If you use other elements from wp.element like Fragment, then the pragma config must be there also: "pragmaFrag": "wp.element.Fragment"

"presets": [
    "@babel/preset-env",
    [
      "@babel/preset-react",
      {
        "pragma": "wp.element.createElement"
      }
    ]
  ]

Here is a reference https://css-tricks.com/learning-gutenberg-6-setting-up-a-custom-webpack-config/